softwerkskammer/locales/translation-de.json

```diff
@@ -69,7 +69,12 @@
     "delete": "Löschen",
     "new": "Neu",
     "guest": "Gast",
-    "none": "keine"
+    "none": "keine",
+    "events_in_europe": "Europaweite Veranstaltungen",
+    "events_in_europe_desc": "Jahreskalender von Konferenzen, Open Spaces, BarCamps und anderen Veranstaltungen",
+    "socrates_text": "5. Internationale Software Craftsmanship and Testing Conference",
```

Comment:
Das ist aber ziemlich Denglish. Ich würde hier das rein englische bevorzugen anstelle von ...internationale...and...conference